README.md

```diff
@@ -60,33 +60,39 @@ The Proxies have to fetch certificates from the central Certificate Authority, h
 
 ## Getting started
 
+If you have docker installed and cloned our repository you can run a demo beam network using the following command:
+```bash
+./dev/beamdev demo
+```
+This will launch a beam network consisting of two proxies (Proxy1 localhost:8081 and Proxy2 localhost:8082) and a broker (localhost:8080)
```

Comment:
```suggestion
This will launch your own beam demo network, which consists of one broker (listening on `localhost:8080`) and two connected proxies (listening on `localhost:8081` and `localhost:8082`).
```